compiler/rustc_public/src/compiler_interface.rs

```diff
@@ -835,6 +835,8 @@ impl<'tcx> CompilerInterface<'tcx> {
 // A thread local variable that stores a pointer to [`CompilerInterface`].
 scoped_tls::scoped_thread_local!(static TLV: Cell<*const ()>);
 
+// FIXME(makai410): remove this cfg when we have a stable driver.
```

Comment:
I don't think we need this FIXME... Maybe just a comment. We could also move this function to be inside `rustc_internal`.